src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyTestSwitchExpressionsNewLine.java

```diff
@@ -14,8 +14,9 @@
 //non-compiled with javac: Compilable with Java14
 package com.puppycrawl.tools.checkstyle.checks.blocks.leftcurly;
 
-public class InputLeftCurlyTestSwitchExpressionsNewLine { // violation
-    int howMany1(int k) { // violation
+// violation below ''{' at column 57 should be on a new line'
```

Comment:
Please make comment on line 16, in this case you do not need to shift all other violations in expected array.

Reply:
Done